### PR TITLE
am, bugreport, bugreportz, cmd, dalvikvm, dumpsys: add Polish translation

### PR DESCRIPTION
**Pull request by kruczyna**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

If you guys prefer single PR's per translation page, I can close this one and issue 6 separate ones, just let me know 🙏

### CODE REVIEW

File: pages.pl/android/dalvikvm.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# dalvikvm
+
+> Wirtualna maszyna Android Java.
+> Więcej informacji: <https://source.android.com/devices/tech/dalvik>.
+
+- Uruchom program Java:
+
+`dalvikvm -classpath {{path/to/file.jar}} {{classname}}`
```
Comment (kbdharun) [inline pages.pl/android/dalvikvm.md:8]:
```suggestion
`dalvikvm -classpath {{ścieżka/do/pliku.jar}} {{classname}}`
```
[Reference](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md)

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution. I have a small suggestion for this page.

File: pages.pl/android/dalvikvm.md (hunk 1 above)
Comment (kruczyna) [reply pages.pl/android/dalvikvm.md:8]: Of course! On it :)

**Review by kbdharun — APPROVED** [review]

File: pages.pl/android/dalvikvm.md (hunk 1 above)
Comment (kbdharun) [reply pages.pl/android/dalvikvm.md]: The current translation looks fine. Also, we don't translate intent classes and other arguments. We translate simple command arguments like {{class_name}},{{file_name}},etc to their native equivalents also not all arguments require translation in some cases we prefer the English word.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**